x-pack/plugins/apm/server/lib/services/get_throughput.ts

```diff
@@ -28,6 +28,9 @@ interface Options {
 type ESResponse = PromiseReturnType<typeof fetcher>;
 
 function transform(response: ESResponse) {
+  if (response.hits.total.value === 0) {
+    return [];
+  }
   const buckets = response.aggregations?.throughput?.buckets ?? [];
```

Comment:
This should be `response.aggregations?.throughput.buckets`. If `aggregations` is defined, so should `throughput`.